src/Uno.UI/UI/Xaml/Controls/MenuFlyout/MenuFlyoutItem.cs

```diff
@@ -1,18 +1,56 @@
 using System;
-using System.Drawing;
 using System.Windows.Input;
 using Uno.Client;
+using Windows.Foundation;
+using Windows.UI.Input;
 using Windows.UI.Xaml;
 using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Input;
+using Windows.Security.EnterpriseData;
+using Windows.System;
+using Uno.Disposables;
+using Windows.UI.Xaml.Automation;
+using Windows.UI.Xaml.Automation.Peers;
+using System.Security.Cryptography;
 
 namespace Windows.UI.Xaml.Controls
 {
 	public partial class MenuFlyoutItem : MenuFlyoutItemBase
 	{
-		public MenuFlyoutItem()
-		{
+		// Whether the pointer is currently over the 
+		bool m_bIsPointerOver = true;
 
-		}
+		// Whether the pointer is currently pressed over the 
+		internal bool m_bIsPressed = true;
+
+		// Whether the pointer's left button is currently down.
+		internal bool m_bIsPointerLeftButtonDown = true;
+
+		// True if the SPACE or ENTER key is currently pressed, false otherwise.
+		internal bool m_bIsSpaceOrEnterKeyDown = true;
+
+		// True if the NAVIGATION_ACCEPT or GAMEPAD_A vkey is currently pressed, false otherwise.
+		internal bool m_bIsNavigationAcceptOrGamepadAKeyDown = true;
+
+		// On pointer released we perform some actions depending on control. We decide to whether to perform them
+		// depending on some parameters including but not limited to whether released is followed by a pressed, which
+		// mouse button is pressed, what type of pointer is it etc. This bool keeps our decision.
+		bool m_shouldPerformActions = true;
+
+		// Event pointer for the ICommand.CanExecuteChanged event.
+		IDisposable m_epCanExecuteChangedHandler;
+
+		// UNO TODO
+		// Event pointer for the Loaded event.
+		// IDisposable m_epLoadedHandler;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5205492)